tasks/deploy/factory.ts

```diff
@@ -5,21 +5,27 @@ import { AirbroCampaignFactory } from "../../src/types/contracts/AirbroCampaignF
 import { AirbroCampaignFactory__factory } from "../../src/types/factories/contracts/AirbroCampaignFactory__factory";
 
 import { Signer } from "@ethersproject/abstract-signer";
+import { isAddress } from "ethers/lib/utils";
 
 task("deploy:AirbroFactory").setAction(async function (taskArguments: TaskArguments, { ethers }) {
   const BACKEND_WALLET_ADDRESS: string | undefined = process.env.BACKEND_WALLET_ADDRESS;
   if (BACKEND_WALLET_ADDRESS === undefined || BACKEND_WALLET_ADDRESS === "") {
     throw new Error("Please define the BACKEND_WALLET_ADDRESS in your .env file.");
   }
+  const TREASURY_WALLET_ADDRESS: string | undefined = process.env.TREASURY_WALLET_ADDRESS;
+  if (TREASURY_WALLET_ADDRESS === undefined || TREASURY_WALLET_ADDRESS === "") {
+    throw new Error("Please define the TREASURY_WALLET_ADDRESS in your .env file.");
+  }
 
   const accounts: Signer[] = await ethers.getSigners();
   console.log("Deployer address: " + (await accounts[0].getAddress()));
 
   const AirbroCampaignFactoryFactory: AirbroCampaignFactory__factory = <AirbroCampaignFactory__factory>(
     await ethers.getContractFactory("AirbroCampaignFactory")
   );
+  // for some reason TREASURY_WALLET_ADDRESS here is displayed as an error
```

Comment:
yarn typechain?

Reply:
looks like that was it, it's not red anymore